Header header
uint8 nodeId
uint8 port
uint8[] data